common-app-lib/src/cmr/common_app/services/search/query_model.clj

```diff
@@ -228,6 +228,12 @@
 (defrecord MatchNoneCondition
   [])
 
+(defrecord MatchCondition
+  [field value])
+
+(defrecord MatchFilterCondition
+  [field value filter])
+
```

Comment:
Could you add a comment for each argument in the defrecord that describes what it is. Similar to the other Conditions in that namespace.

Reply:
updated